packages/spacecat-shared-rum-api-client/src/common/rum-bundler-client.js

```diff
@@ -190,11 +190,13 @@ async function fetchBundles(opts = {}) {
   for (const chunk of chunks) {
     const responses = await Promise.all(chunk.map((url) => fetch(url)));
     const bundles = await Promise.all(responses.map((response) => response.json()));
-    result.push(...bundles.flatMap(
-      (b) => b.rumBundles.filter(
-        (bundle) => !filterBotTraffic || !isBotTraffic(bundle),
-      ).map(filterEvents(checkpoints)),
-    ));
+
+    bundles.forEach((b) => {
+      b.rumBundles
+        .filter((bundle) => !filterBotTraffic || !isBotTraffic(bundle))
+        .map(filterEvents(checkpoints))
+        .forEach((filteredEvent) => result.push(filteredEvent));
```

Comment:
Can we call this (filteredEvent) as filteredBundle as that's a bundle we're adding to the result?